components/apimgt/org.wso2.carbon.apimgt.core/src/main/java/org/wso2/carbon/apimgt/core/exception/ApiStoreSdkGenerationException.java

```diff
@@ -0,0 +1,42 @@
+/***********************************************************************************************************************
+ * *
+ * *   Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * *
+ * *   WSO2 Inc. licenses this file to you under the Apache License,
+ * *   Version 2.0 (the "License"); you may not use this file except
+ * *   in compliance with the License.
+ * *   You may obtain a copy of the License at
+ * *
+ * *     http://www.apache.org/licenses/LICENSE-2.0
+ * *
+ * *  Unless required by applicable law or agreed to in writing,
+ * *  software distributed under the License is distributed on an
+ * *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * *  KIND, either express or implied.  See the License for the
+ * *  specific language governing permissions and limitations
+ * *  under the License.
+ * *
+ */
```

Comment:
There are double * marks in the licence header. This will happen if you copy the licence header inside a /* */. Please correct this. You can use IDE settings to attach a licence header automatically when creating a new file. I see several places,  please correct this in all the files.